### PR TITLE
As a potential customer I want to have a free consultation form, so that I can evaluate if they are right partners me.

### PR DESCRIPTION
**Pull request by lspk129**
https://tree.taiga.io/project/troublemaker-website/us/5

### CODE REVIEW

File: components/Contact/Components/RaisedButtonStyled/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import RaisedButton from 'material-ui/RaisedButton';
+import styled from 'styled-components';
+
+const RaisedButtonStyled = styled(RaisedButton)`
+  // margin-top: 30px;
```
Comment (tmxyz) [inline components/Contact/Components/RaisedButtonStyled/index.js]: delete this

File: components/Contact/StyledColumn/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+import { Column } from 'hedron';
+
+const StyledColumn = styled(Column)`
+  background: #f0f0f0;
```
Comment (tmxyz) [inline components/Contact/StyledColumn/index.js]: No hardcoded colors.

File: components/Contact/index.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+import muiThemeable from 'material-ui/styles/muiThemeable';
+import { Row, Column } from 'hedron';
+
+import ContactTitle from './ContactTitle';
+import RaisedButtonStyled from './Components/RaisedButtonStyled';
+import Paragraph from './Components/Paragraph';
+import PaperIcons from './Components/PaperIcons';
+import FontAwesomeIcon from './Components/FontAwesomeIcon';
+import StyledColumn from './StyledColumn';
+
```
Comment (tmxyz) [inline components/Contact/index.js]: No extra lines.

File: components/Contact/index.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                  value={this.state.value}
+                  onChange={this.handleChange}
+                  fullWidth
+                >
+                  <MenuItem value={1} primaryText="I want an introduction to your services..." />
+                  <MenuItem value={2} primaryText="I want a free price quote..." />
+                  <MenuItem value={3} primaryText="I want to ask a question" />
+                </SelectField>
+                {this.state.value === 3 && (
+                <TextField
```
Comment (tmxyz) [inline components/Contact/index.js]: Indentation

File: components/Contact/Components/RaisedButtonStyled/index.js (hunk 1 above)
Comment (lspk129) [inline components/Contact/Components/RaisedButtonStyled/index.js]: Fixed.

File: components/Contact/StyledColumn/index.js (hunk 2 above)
Comment (lspk129) [inline components/Contact/StyledColumn/index.js]: Fixed.

File: components/Contact/index.js (hunk 3 above)
Comment (lspk129) [inline components/Contact/index.js]: Fixed.

File: components/Contact/index.js (hunk 4 above)
Comment (lspk129) [inline components/Contact/index.js]: Fixed.

**Pull request merged**